templates/20years/index.html

```diff
@@ -0,0 +1,1402 @@
+              </p>
+            </div>
+            <div class="col-3">
+              <p class="u-no-margin--bottom">
+                <strong>Andrew Wafaa</strong>
+              </p>
+              <p class="u-text--muted">Senior Director and Fellow, Arm</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="p-section">
+      <div class="row--25-75">
+        <div class="col u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/b93ffef5-Dell%20Technologies%20Logo.png"
+               alt="Dell logo"
+               width="126"
+               height="50"
+               style="margin-top:1rem" />
+        </div>
+        <div class="col">
+          <hr class="is-muted" />
+          <div class="row">
+            <div class="col-6">
+              <p>
+                <i>"Congratulations on Ubuntu's 20th anniversary! I am honored to acknowledge our long-standing partnership with Canonical. Together, we have brought innovative solutions to our customers, addressing their unique needs through the power of open source. We look forward to many more years of continued success."</i>
+              </p>
+            </div>
+            <div class="col-3">
+              <p class="u-no-margin--bottom">
+                <strong>Dan Mroz</strong>
+              </p>
+              <p class="u-text--muted">C4 Chief Cloud Strategists, Dell</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="p-section">
+      <div class="row--25-75">
+        <div class="col u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/0600e0fa-Hewlett%20Packard%20Enterprise%20Logo.png"
+               alt="HP Enterprise logo"
+               width="126"
+               height="50"
+               style="margin-top:1rem" />
+        </div>
+        <div class="col">
+          <hr class="is-muted" />
+          <div class="row">
+            <div class="col-6">
+              <p>
+                <i>"We're proud to be a longstanding collaborator with Canonical. Starting with Ubuntu certification on HPE platforms, we expanded to meet customer needs for open source clouds. As Canonical celebrates 20 years, we look forward to deepening our partnership to deliver cloud modernization and AI initiatives to customers."</i>
+              </p>
+            </div>
+            <div class="col-3">
+              <p class="u-no-margin--bottom">
+                <strong>Tam Vuong</strong>
+              </p>
+              <p class="u-text--muted">Technology Partner Program Lead, HPE (Hewlett Packard Enterprise)</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow u-fixed-width">
+      <hr class="p-rule" />
```

Comment:
So there was a pretty conversation around this in [this pr](https://github.com/canonical/vanilla-framework/pull/5241) I made in Vanilla. And the consensus was we should be using 'p-rule' basically everywhere for consistency with the other 'p-rule' variants. But maybe I missed something?

Reply:
Actually I've been leaving this comment based on this exact pr 😅 From the rule notes in releases.yaml: "We've deprecated the <code>.p-rule</code> component. Use the <a href="/docs/base/separators#no-bottom-margin">HR with no bottom margin</a> instead." Is this stale?

Reply:
I am just going with what it says on the [rule pattern](https://vanillaframework.io/docs/patterns/rule#default). But there is also the [separators](https://vanillaframework.io/docs/base/separators) which seems to say something different. So I don't know. But if we are following BEM conventions, it would be the former